### ISSUE TITLE
LOG(WARN) on options breaks O2 DPL error reporting

### ISSUE THREAD

**Issue by dberzano** [FairRootGroup/FairMQ#52]
`fairmq/options/FairMQProgOptions.cxx` now has a line:

```c++
LOG(WARN) << "detected unregistered option: " << o;
```

This causes the O2 DPL to mark all running devices in red due to the presence of the warning message (@ktf: by the way FairMQ v1.2.5 fixes the old WARN problem but introduces this).

An example of error messages is:

```
[46204]: [17:27:54][WARN] detected unregistered option: --num-files
[46204]: [17:27:54][WARN] detected unregistered option: 1
[46204]: [17:27:54][WARN] detected unregistered option: --num-workers
[46204]: [17:27:54][WARN] detected unregistered option: 4
[46204]: [17:27:54][WARN] detected unregistered option: --sleep
[46204]: [17:27:54][WARN] detected unregistered option: 1
```

This is due to the way DPL forwards command-line options to the underlying devices, I don't know if there's a proper way to tell FairMQ to simply ignore them?

Or maybe we should fix it somehow on our side?

Thanks!

**Comment by rbx** [comment]
I guess I could make this one also DEBUG, you guys really don't like warnings ;)

**Comment by dberzano** [comment]
I think it should be a debug message, not a runtime warning. The problem is that every time we have a `LOG(WARN)` the DPL interface has boxes turning orange, see below:

![schermata 2018-06-14 alle 12 59 18](https://user-images.githubusercontent.com/825677/41408411-e10b4eac-6fd2-11e8-90c7-ea4564d27661.png)

In practice we allow users to use `LOG(WARN)` or `LOG(ERROR)` to point out that something's wrong in the user's code and we signal the problem with a visible color.

**Comment by rbx** [comment]
Fixed with #53.
